test-common/utils.c

```diff
@@ -2932,12 +2932,17 @@ eos_test_client_prepare_volume (EosTestClient *client,
                                                                                "eos-updater-prepare-volume",
                                                                                "eos-updater-prepare-volume",
                                                                                NULL);
+  g_autofree gchar *libeos_updater_util_path = g_test_build_filename (G_TEST_DIST,
```

Comment:
Shouldn’t this be s/`G_TEST_DIST`/`G_TEST_BUILT`/, since the typelib is generated at build time, rather than shipped in git?